tests/main/interfaces-snap-interfaces-requests-control/task.yaml

```
@@ -18,10 +18,16 @@ environment:
     PYTHONIOENCODING: utf-8
 
 prepare: |
+    if not tests.info is-reexec-in-use && os.query is-ubuntu 22.04; then
+        tests.exec skip-test "Ubuntu 22.04 kernel doesn't support prompting" && exit 0
+    fi
```

Any reason this is different from the checks from other tests?

```suggestion
    if os.query is-ubuntu 20.04; then
        tests.exec skip-test "Ubuntu 20.04 kernels doesn't support prompting" && exit 0
    fi
    if not tests.info is-reexec-in-use && os.query is-ubuntu 22.04; then
        tests.exec skip-test "Ubuntu 22.04 AppArmor parser doesn't support prompting" && exit 0
    fi
```